expression/builtin.go

```diff
@@ -33,7 +33,8 @@ type baseBuiltinFunc struct {
 	deterministic bool
 }
 
-func newBaseBuiltinFunc(args []Expression, ctx context.Context) baseBuiltinFunc {
+// newBaseBuiltinFunc creates a new BaseBuildFunc.
+func newBaseBuiltinFunc(ctx context.Context, args []Expression) baseBuiltinFunc {
```

Comment:
Alright, I use VS Code to dev and return the issues.

Forget it, I'll revert them back.